lib/capybara/node/actions.rb

```diff
@@ -70,28 +64,25 @@ def click_button(locator = nil, **options)
       #     el.fill_in with: 'Tom'
       #
       #
-      # @overload fill_in([locator], with:, **options)
-      #   @param [String] locator                 Which field to fill in
-      #   @param [Hash] options
-      #   @param with: [String]                  The value to fill_in
-      #   @macro waiting_behavior
-      #   @option options [String] currently_with The current value property of the field to fill in
-      #   @option options [Boolean] multiple      Match fields that can have multiple values?
-      #   @option options [String, Regexp] id             Match fields that match the id attribute
-      #   @option options [String] name           Match fields that match the name attribute
-      #   @option options [String] placeholder    Match fields that match the placeholder attribute
-      #   @option options [String, Array<String>, Regexp] class    Match fields that match the class(es) provided
-      #   @option options [Hash] fill_options     Driver specific options regarding how to fill fields (Defaults come from Capybara.default_set_options)
-      #
-      # @return [Capybara::Node::Element]  The element filled_in
-      def fill_in(locator = nil, with:, currently_with: nil, fill_options: {}, **find_options)
-        find_options[:with] = currently_with if currently_with
-        find_options[:allow_self] = true if locator.nil?
-        find(:fillable_field, locator, find_options).set(with, fill_options)
+      # @param [String] locator                    Which field to fill in
```

Comment:
Why the change?

Reply:
Because I think that to use the original method signature is more simple and DRY than using `@overload`.
And I think that to use `@param` for keyword arguments is better than `@options` according to the [YARD advice](https://www.rubydoc.info/gems/yard/file/docs/Tags.md#option) below.

> Note: For keyword parameters, use `@param`, not `@option`.

But this is also too much for this PR. 😓